src/Stack/Package.hs

```diff
@@ -77,6 +77,7 @@ import           System.IO.Error
 import           RIO.Process
 import           RIO.PrettyPrint
 import qualified RIO.PrettyPrint as PP (Style (Module))
+import Stack.Types.Dependency (DepValue(..), DepType (..))
```

Comment:
Please could this `import` line adopt the same format as used elsewhere in the module.